tests/pthread/test_pthread_asan_use_after_free_2.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  var output = [];
+  Module['printErr'] = function (text) {
+    if (text == '==42==ABORTING') {
+      var result = output.join('\n');
+      var passed = [
+        'ERROR: AddressSanitizer: heap-use-after-free on address',
+        'READ of size 4',
+        'is located 0 bytes inside of 4-byte region',
+        'freed by thread T1 here:',
+        'previously allocated by thread T1 here:',
+        'Thread T1 created by T0 here:',
+        'SUMMARY: AddressSanitizer: heap-use-after-free',
+        'Shadow bytes around the buggy address:',
+        'Shadow byte legend (one shadow byte represents 8 application bytes):',
```

Comment:
When these tests fail its really hard to know why.. I wonder if we can pipe the stderr back to the server and use the same output comparison methods we use in non-browser tests?

Do you know how common this pattern is in browser tests?  Is it just the sanitizer tests?

Reply:
I think I've only seen this pattern in the sanitizer tests. Yeah, I agree it's a little inconvenient, +1 for looking into possible improvements.